src/modules/networking/src/lib/Networking.h

```diff
@@ -47,6 +47,13 @@ struct NetworkingSettings
     std::string connected;
 };
 
+enum class NetworkManagementService
```

Comment:
Instead of a scoped to class enumeration declared outside of the class scope, better to define such enum as a class member. Same for NetworkingSettingType below.